pkg/apis/core/validation/validation.go

```diff
@@ -2824,6 +2824,17 @@ func validateDNSPolicy(dnsPolicy *core.DNSPolicy, fldPath *field.Path) field.Err
 	return allErrors
 }
 
+func validateFSGroupChangePolicy(fsGroupPolicy *core.PodFSGroupChangePolicy, fldPath *field.Path) field.ErrorList {
+	allErrors := field.ErrorList{}
+	validValues := sets.NewString(string(core.AlwaysChangeVolumePermission), string(core.OnRootMismatch))
+
+	if !validValues.Has(string(*fsGroupPolicy)) {
+		validValues := []string{string(core.AlwaysChangeVolumePermission), string(core.OnRootMismatch)}
```

Comment:
sets have a `.List()` - see how `supportedPortProtocols` is handled

Reply:
*headdesk*